netbox/templates/ipam/aggregate.html

```diff
@@ -40,6 +40,14 @@
     </div>
     <h1>{% block title %}{{ aggregate }}{% endblock %}</h1>
     {% include 'inc/created_updated.html' with obj=aggregate %}
+    {% if show_available is not None %}
```

Comment:
Can you move this code to a separate template (e.g. `ipam/inc/toggle_available.html`) and use an `{% include %}` tag to avoid the repetition?